src/extension/index.d.ts

```diff
@@ -9,8 +9,9 @@ export interface Api {
      * @param logs An array of Uris to open.
      */
     openLogs(logs: Uri[]): Promise<void>;
-    closeLogs(logs: Uri[]): Promise<void>;
-    closeAllLogs(): Promise<void>;
-    selectByIndex(uri: Uri, runIndex: number, resultIndex: number): Promise<void>;
+    closeLogs(logs: Uri[]): void;
```

Comment:
This is the public API used by other extensions integrating with the SARIF Viewer - and as such might be a breaking change. I think if extensions are using `await closeAllLogs()`, they'll be fine, but if they're doing `closeAllLogs().then(/* ... */)` that will start breaking.

Reply:
Ah...thanks for letting me know. I didn't realize that this was a public facing API used by other extensions. I can undo these changes to ensure compatibility. I'll also add a comment so that we don't make any accidental changes in the future.